src/UI/Root/PrimaryPageManager.tsx

```diff
@@ -27,6 +29,7 @@ import { ServiceInstanceHistoryPage } from "@S/ServiceInstanceHistory/UI";
 import { ServiceInventoryPage } from "@S/ServiceInventory/UI";
 import { SettingsPage } from "@S/Settings/UI";
 import { StatusPage } from "@S/Status/UI";
+import { features } from "../../config";
```

Comment:
If undefined this might throw an error

Reply:
what do you propose?

Reply:
`import * as configFile from "../../config";`

Something like this. The file will always be there, it's the content that can be variable. To make sure you are not trying to import an un-existing variable, you can then do the check : `Object.hasOwn(configFile, "property")` or any other way you'd prefer to make sure it is valid before trying to access sub-properties.